src/main/resources/de/qabel/desktop/ui/accounting/menuqr/menuqr.fxml

```diff
@@ -0,0 +1,83 @@
+            xmlns:fx="http://javafx.com/fxml/1" fx:controller="de.qabel.desktop.ui.accounting.menuqr.MenuQRController">
+    <children>
+        <VBox fx:id="vboxMenu" styleClass="vbox-menu">
+            <HBox>
+                <children>
+                    <Label styleClass="icon-edit">
+                        <HBox.margin>
+                            <Insets left="5" bottom="3" top="3"/>
+                        </HBox.margin>
+                    </Label>
+                    <Button fx:id="labelEdit" text="%editDetails" styleClass="button-menu"/>
+                </children>
+            </HBox>
+            <HBox>
+                <children>
+                    <Label styleClass="icon-delete">
+                        <HBox.margin>
+                            <Insets left="5" bottom="3" top="3"/>
+                        </HBox.margin>
+                    </Label>
+                    <Button fx:id="labelDelete" wrapText="true" text="%removeIdentity" styleClass="button-menu"/>
+
+                </children>
+            </HBox>
+            <Line endX="00.0" startX="-220.0"/>
+            <HBox>
+                <children>
+                    <Label styleClass="icon-export">
+                        <HBox.margin>
+                            <Insets left="5" bottom="3" top="3"/>
+                        </HBox.margin>
+                    </Label>
+                    <Button fx:id="labelExport" wrapText="true" text="%exportIdentityQID" styleClass="button-menu"/>
+                </children>
+            </HBox>
+            <HBox>
+                <children>
+                    <Label styleClass="icon-qrcode">
+                        <HBox.margin>
+                            <Insets left="5" bottom="3" top="3"/>
+                        </HBox.margin>
+                    </Label>
+                    <Button fx:id="labelPrivateKey" onAction="#openQRCode" text="%exportPrivateKeyQR"
+                            styleClass="button-menu"/>
+                </children>
+            </HBox>
+            <Line endX="00.0" startX="-220.0"/>
+            <HBox>
+                <children>
+                    <Label styleClass="icon-qrcode">
+                        <HBox.margin>
+                            <Insets left="5" bottom="3" top="3"/>
+                        </HBox.margin>
+                    </Label>
+                    <Button fx:id="labelPublicKeyQR" wrapText="true" text="%sharePublicKeyQR" styleClass="button-menu"/>
+                </children>
+            </HBox>
+            <HBox>
+                <children>
+                    <Label styleClass="icon-email">
+                        <HBox.margin>
+                            <Insets left="5" bottom="3" top="3"/>
+                        </HBox.margin>
+                    </Label>
+                    <Button fx:id="labelPublicKeyEmail" wrapText="true" text="%sharePublicKeyEmail"
```

Comment:
All these buttons look good but are useless (no `onAction`). We can't use this in production